cloudpickle/cloudpickle.py

```diff
@@ -1362,3 +1378,15 @@ def _get_bases(typ):
         # For regular class objects
         bases_attr = '__bases__'
     return getattr(typ, bases_attr)
+
+
+def _make_dict_keys(obj):
+    return dict.fromkeys(obj).keys()
```

Comment:
I am not sure it's worth rebuilding a fake dict with `None` valued values. Maybe we could just ship the list of keys and values directly?

Reply:
That's a good point.